x/nft/blockchain/init.go

```diff
@@ -25,8 +25,8 @@ func (i *Initializer) FromGenesis(opts weave.Options, db weave.KVStore) error {
 }
 
 type genesisBlockchainToken struct {
-	ID    string `json:"id"`
-	Owner string `json:"owner"`
+	ID    weave.Address `json:"id"`
```

Comment:
I don't think `ID` should be a `weave.Address`. This is a ASCII string (binary, but readable) and [soon it should be a string](https://github.com/iov-one/weave/issues/209). So if weave.Address is needed for implementation, I would keep it where it was.